### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by taleldayekh**
**Thank you for submitting this pull request :)**

Fixes # 

**Short description**

**Definition of Done**

- [ ] Steps to manually test the change have been documented
- [ ] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

**Comment by andrevmatos** [comment]
I managed to workaround the issues with ts-jest: https://github.com/kulshekhar/ts-jest/issues/1604
But let's wait a little before merging, maybe the workarounds can be reverted and we get a simpler setup with up-to-date dependencies.

File: raiden-ts/src/channels/utils.ts [hunk 1]
```diff
@@ -91,6 +91,12 @@ export function assertTx(
   error: ErrorCodes,
   { log }: Pick<RaidenEpicDeps, 'log'>,
 ): OperatorFunction<ContractTransaction, Hash> {
+  /**
+   * Operator to check for tx
+   *
+   * @param tx - pending contract tx
+   * @returns Observable of txHash
+   */
```
Comment (nephix) [inline raiden-ts/src/channels/utils.ts:99]: Call me crazy but this just seems wrong 😅

Comment (andrevmatos) [reply raiden-ts/src/channels/utils.ts:99]: Only way I could make `eslint` happy. Seems they changed something recently regarding those functions which return functions. It's ugly, but makes sense to have docstrings for outer and inner functions.

Comment (nephix) [reply raiden-ts/src/channels/utils.ts:99]: No it doesn't make any sense at all 😅

**Comment by kelsos** [comment]
> Had to pin `jest` & `ts-jest`, but could get rid of the workarounds until that issue get properly fixed and `ts-jest` with official `jest@26` support is released. I think ts-jest is still pinned to jest@24. :P

**Pull request merged**